central/complianceoperator/v2/scansettingbindings/datastore/datastore_impl.go

```diff
@@ -0,0 +1,34 @@
+package datastore
```

Comment:
I would prefer a test file.  But this file doesn't do anything but call the store.  🤷